### PR TITLE
Completed javascript-if exercise-Chris Mendoza

### PR DESCRIPTION
**Pull request by CTMendoza**
javascript-if

### CODE REVIEW

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if (!person.age >= 18 && !person.age >= 21) {
```
Comment (BrettAlbright) [inline javascript-if/main.js]: this one should just return false, since it is always illegal

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny update!

**Comment by CTMendoza** [comment]
Updated

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**